Shared/Samples/Show callout/ShowCalloutView.swift

```diff
@@ -40,8 +45,8 @@ struct ShowCalloutView: View {
                     Text("Location")
                         .font(.headline)
                     Text(
-                        CoordinateFormatter.toLatitudeLongitude(
-                            point: callout.location,
+                        CoordinateFormatter.latitudeLongitudeString(
```

Comment:
This is a recent change in the API. In order to make this branch build I included it in this PR. Let me know if you want it in a separate PR.

Reply:
It would be better to create a PR branch with fixes for API changes and then base this branch off of that one. That way, those fixes get in sooner since this PR will take longer to review.